### DISCUSSION TITLE
Operating Systems and Utilites

### DISCUSSION THREAD

**Discussion by Mightyretro** [antonym/netboot.xyz#1064] [General]
Hey guys, i have few suggestions you can add to your compability list :)

Raspberry Pi OS
https://www.raspberrypi.org/

Pfsense
https://www.pfsense.org/

TrueNAS
https://www.truenas.com/

XCP-ng
https://xcp-ng.org/

Passcape Reset Windows Password
https://www.passcape.com/

Kon-Boot
https://kon-boot.com/

Rescuezilla
https://rescuezilla.com/

R-Studio Emergency GUI
https://www.r-studio.com/downloads/RStudioEmg8.iso

batocera
https://batocera.org/

Lakka
https://www.lakka.tv/

Bliss OS (Based on Android)
https://blissos.org/

volumio
https://volumio.com/en/

// Best Regards :)

**Comment by davidbuzz** [comment]
+1 for xcp-ng, its in the same category as proxmox , infact its been around longer and is better.
